packages/import/src/importers/jiraCsv/JiraCsvImporter.ts

```diff
@@ -83,6 +86,7 @@ export class JiraCsvImporter implements Importer {
       const release = row.Release && row.Release.length > 0 ? `Release: ${row.Release}` : undefined;
       const assigneeId = row.Assignee && row.Assignee.length > 0 ? row.Assignee : undefined;
       const status = row.Status;
+      const estimate = row[estimateCustomField] ? parseInt(row[estimateCustomField]) : undefined;
```

Comment:
Do we care to check this is not `NaN`, here and in the other spots – probably good to be defensive?